pkg/agent/querier/querier.go

```diff
@@ -193,12 +193,15 @@ func (aq agentQuerier) GetAgentInfo(agentInfo *v1beta1.AntreaAgentInfo, partial
 		agentInfo.Version = querier.GetVersion()
 		agentInfo.PodRef = querier.GetSelfPod()
 		agentInfo.NodeRef = querier.GetSelfNode(true, aq.nodeConfig.Name)
+		// Make a new string slice instead of appending agentInfo.NodeSubnet directly to avoid duplicate CIDRs.
+		nodeSubnet := make([]string, 0)
```

Comment:
rename it as nodeSubnets?

Reply:
I use nodeSubnet here to make it consistent with CRD field NodeSubnet. But it may be better to change both of them to NodeSubnets as we support dual stack now. Let me try to update this part.

Reply:
Done